tests/test_validation.py

```diff
@@ -10,63 +10,65 @@
 from schematools.types import Json, TableVersions
 from schematools.utils import dataset_schema_from_path
 from schematools.validation import (
-    ActiveVersionsValidator,
-    IdentPropRefsValidator,
-    PsqlIdentifierLengthValidator,
-    ValidationError,
-    _validate_camelcase,
+    _active_versions,
+    _camelcase_ident,
+    _identifier_properties,
+    _postgres_identifier_length,
+    run_all,
```

Comment:
perhaps `from schematools import validation` so it's more obvious that when calling those private functions it's ovious they don't sit in this file.

Reply:
I have an even better idea. I changed the tests to use `run_all` (renamed `run`; maybe just call it `validate`?) instead of the specific private functions. That should give better test coverage.